extensions/dicom-seg/README.md

```diff
@@ -0,0 +1 @@
+# @ohif/extension-dicom-html
```

Comment:
typo

Reply:
^ What Erik said

May be worth copying and updating the basic table we have in the `cornerstone` extension. We should provide some high level information about what's contained in the project. This is also used as the "info" for the npm registry.

Reply:
Update. Now it has a short description of the extension and the tool configuration section. The other sections don't apply for now. @dannyrb if there's anything else I could add, just let me know